extensions/wikia/Piggyback/Piggyback.php

```diff
@@ -30,7 +30,6 @@
 $wgSpecialPages['Piggyback'] = 'Piggyback'; # Let MediaWiki know about your new special page.
 
 $wgSpecialPageGroups['Piggyback'] = 'users';
-$wgLogRestrictions['piggyback'] = 'piggyback';
```

Comment:
I'd consider leaving it in the extension anyway. Can't hurt for the duplication and makes sure it's obvious this is the case when looking at the code of the extension, otherwise could be confusing/concerning as to whether this is restricted.